README.md

```diff
@@ -6,52 +6,66 @@ Try the current version at: [https://cs121-team-panda.github.io/coconut-interpre
 
 Developed by Jonathan Cruz, Teerapat Jenrungrot, Natalie Kadonaga, and Brittany Wang.
 
-## Installation
-```
-pip install -r requirements.txt
-```
+## Architecture 
+<img width="1093" alt="architecture" src="https://user-images.githubusercontent.com/35832643/38783767-b5ef5236-40bb-11e8-91b4-e1d5bdc0aa18.png">
 
-## Running the app
+The Coconut Online Interpreter project's architecture design is based on server-less computing based on AWS Lambda for executing user-submitted Coconut codes. The front-end of the project is developed using React. Based on the designed architecture shown above, the interpreter is resilient to any malicious code submitted by users since the backend protection is handled by AWS Lambda services.
+
+### Prerequisites
 Requires Python 3. 
```

Comment:
Say that the following command is to run the server locally. Perhaps, you should mention the command for testing, as well.

Reply:
For testing, do you mean the `yarn install` and `yarn start` commands?